Header header
gaussian_generator/SingleGaussian[] gaussian ## all of gaussian
int32 size # gaussian image size
